lib/api/bucketPutACL.js

```diff
@@ -44,6 +45,12 @@ const monitoring = require('../utilities/monitoringHandler');
 function bucketPutACL(authInfo, request, log, callback) {
     log.debug('processing request', { method: 'bucketPutACL' });
 
+    const err = validateChecksumsNoChunking(request.headers, request.post);
+    if (err && err.error !== ChecksumError.MissingChecksum) {
+        log.debug(err, { error: errors.BadDigest });
```

Comment:
I think the debug can be simplified into
```suggestion
        log.debug(err.error, err.details);
```

as the BadDigest will show in the response already

Reply:
Does the BadDigest get logged after we exit the handler? If not how do will we know that we sent a BadRequest to the client?

Reply:
I'm not sure, but there might be some debug logs to print the error sent to client

Reply:
Before my changes we logged the BadDigest so I rather keep it

Reply:
removed the BadDigest